src/SFA.DAS.EAS.Portal/Application/Services/AccountDocumentServiceWithSetProperties.cs

```diff
@@ -19,6 +19,11 @@ public Task<AccountDocument> Get(long id, CancellationToken cancellationToken =
             return _accountDocumentService.Get(id, cancellationToken);
         }
 
+        public Task<AccountDocument> GetOrCreate(long id, CancellationToken cancellationToken = default)
```

Comment:
Would it be worth splitting the Get/GetOrCreate and the Save out into separate interfaces to stop duplication of code?